sources/packages/backend/libs/integrations/src/services/sfas/sfas-individual.service.ts

```diff
@@ -94,7 +94,13 @@ export class SFASIndividualService
    * Bulk operation to update student id in SFAS individuals table after importing data from SFAS.
    */
   async updateStudentId(): Promise<void> {
-    await this.repo.manager.query(this.bulkUpdateStudentIdSQL);
+    try {
+      await this.repo.manager.query(this.bulkUpdateStudentIdSQL);
+    } catch (error) {
+      throw new Error("Error while updating ids imported from SFAS.", {
```

Comment:
May be adjusted to be explicit about the ids, for instance, "Error while updating student ids imported from SFAS.".